import_export/resources.py

```diff
@@ -1196,9 +1197,16 @@ def field_from_django_field(cls, field_name, django_field, readonly):
 
         FieldWidget = cls.widget_from_django_field(django_field)
         widget_kwargs = cls.widget_kwargs_for_field(field_name, django_field)
+
+        attribute = field_name
+        column_name = field_name
+        if isinstance(django_field, ForeignKey) and "__" not in column_name:
```

Comment:
A comment referencing the issue could be helpful for future maintainers to understand why the code was added.

Reply:
I have added the comment.